dropwizard-rabbitmq/build.gradle

```diff
@@ -0,0 +1,11 @@
+dependencies {
+    compile project(':dropwizard-guice')
```

Comment:
Do we want to specify a version here?

Reply:
I was just following what the other modules are doing, not sure if there is reasoning behind not specifying a version

Reply:
`dropwizard-guice` is another module in `dropwizard-common`, so you want to use always the latest.